scripts/start.js

```diff
@@ -0,0 +1,19 @@
+import * as dotenv from 'dotenv';
+import Webex from 'webex';
+
+import WebexSDKAdapter from '../src/WebexSDKAdapter';
+
+// Allows convict module to read the corresponding configuration
```

Comment:
Not sure what "convict" means.

Reply:
It's a [tool](https://www.npmjs.com/package/convict) we used in previous projects, prob copy-paste issue